server/service/src/processors/transfer/requisition/mod.rs

```diff
@@ -0,0 +1,190 @@
+pub(crate) mod create_response_requisition;
+pub(crate) mod link_request_requisition;
+pub(crate) mod update_request_requisition_status;
+
+#[cfg(test)]
+pub(crate) mod test;
+
+use repository::{
+    ChangelogAction, ChangelogFilter, ChangelogRepository, ChangelogRow, ChangelogTableName,
+    EqualFilter, KeyValueStoreRepository, KeyValueType, RepositoryError, Requisition,
+    RequisitionFilter, RequisitionRepository, StorageConnection,
+};
+use thiserror::Error;
+
+use crate::{
+    processors::transfer::requisition::{
+        create_response_requisition::CreateResponseRequisitionProcessor,
+        link_request_requisition::LinkeRequestRequisitionProcessor,
+        update_request_requisition_status::UpdateRequestRequstionStatusProcessor,
+    },
+    service_provider::ServiceProvider,
+    sync::{ActiveStoresOnSite, GetActiveStoresOnSiteError},
+};
+
+const CHANGELOG_BATCH_SIZE: u32 = 20;
+
+#[derive(Clone, Debug)]
+pub(crate) struct RequisitionTransferProcessorRecord {
+    requisition: Requisition,
+    linked_requisition: Option<Requisition>,
+    other_party_store_id: String,
+}
+
+#[derive(Error, Debug)]
+pub(crate) enum ProcessRequisitionTransfersError {
+    #[error("Problem getting upsert record {0}")]
+    GetUpsertRecordError(GetUpsertRecordError),
+    #[error("{0}")]
+    GetActiveStoresOnSiteError(GetActiveStoresOnSiteError),
+    #[error("{0:?}")]
+    DatabaseError(RepositoryError),
+    #[error("{0}")]
+    ProcessorError(ProcessorError),
+    #[error("Name id is missing from requisition changelog {0:?}")]
+    NameIdIsMissingFromChangelog(ChangelogRow),
+    #[error("Name is not an active store {0:?}")]
+    NameIsNotAnAciveStore(ChangelogRow),
```

Comment:
```suggestion
    NameIsNotAnActiveStore(ChangelogRow),
```

Reply:
Thanks, also had to update it's use: https://github.com/openmsupply/open-msupply/commit/cdf1c83ffd1f795961007396268e33155aa798e8